src/Logger.cpp

```diff
@@ -161,6 +222,14 @@ bool Logger::WriteDateTime(time_t time) noexcept
 	return logFile.Write(buf.c_str());
 }
 
+bool Logger::WriteLogLevelPrefix(MessageLogLevel messageLogLevel) noexcept
```

Comment:
Can we use one function to write both the datetime and the prefix? do we always write them together?

Reply:
Done